wordpress-mobile/WordPress-iOS
# Updating podfile.lock

Fixes # n/a

To test:
- `rake dependencies`.
- make sure the lock file doesn't change.

PR submission checklist:
